specs/protocol/exec-engine.md

```diff
@@ -164,7 +164,7 @@ Where:
   header of the latest registered L1 origin block.
 
 Conceptually what the above function captures is the formula below, where `compressedTxSize =
-(zeroes*4 + ones*16) / 16` can be thought of as a rough approximation of how many bytes the
+(zeroes*4 + ones*16 + 68*16) / 16` can be thought of as a rough approximation of how many bytes the
```

Comment:
where did 68 come from? seems like magic number

Reply:
I think that's an estimate of the overhead data of a tx, mostly coming from its signature?

Reply:
In some places the 68 is hardcoded, to estimate cost before signing. But after the tx has been signed and encoded there's no need to add the 68. So it completely depends on context whether or not it's a necessary addition.